boards/common/atmega/Makefile.include

```diff
@@ -1 +1,4 @@
 INCLUDES += -I$(RIOTBOARD)/common/atmega/include
+
+# require Makefile.dep for correct dependency management of atmega PCINTs
+include $(RIOTCPU)/$(CPU)/Makefile.dep
```

Comment:
I thin this is the issue with Murdock. I believe the CPU variable is not yet available here. Also: `boards/common/atmega/Makefile.dep` already contains this exact include, so no need to include it twice.